viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts

```diff
@@ -0,0 +1,129 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { MathUtils } from 'three/src/math/MathUtils';
+import { FlexibleControlsType } from './FlexibleControlsType';
+import { FlexibleMouseActionType } from './FlexibleMouseActionType';
+import { FlexibleWheelZoomType } from './FlexibleWheelZoomType';
+
+const DEFAULT_POINTER_ROTATION_SPEED = (0.5 * Math.PI) / 360; // half degree per pixel
+const DEFAULT_KEYBOARD_ROTATION_SPEED = DEFAULT_POINTER_ROTATION_SPEED * 5;
+const DEFAULT_MIN_POLAR_ANGLE = 0.0001;
+
+/**
+ * @beta
+ */
+export class FlexibleControlsOptions {
+  //================================================
+  // INSTANCE FIELDS
+  //================================================
+
+  // Main behaivor
+  public controlsType = FlexibleControlsType.Orbit;
+
+  // Visualization
+  public showTarget = true;
+  public showLookAt = false;
+
+  // Mouse click, double click and wheel behaivor
+  public mouseWheelAction = FlexibleWheelZoomType.Auto;
+  public mouseClickType = FlexibleMouseActionType.None;
+  public mouseDoubleClickType = FlexibleMouseActionType.SetTargetAndCameraPosition;
+  public enableChangeControlsTypeOn123Key = true;
+  public enableKeyboardNavigation = true;
+
+  // Used in FlexibleCameraManager
+  public animationDuration = 300;
+  public minimumTimeBetweenRaycasts = 200; // For mouse wheel event
+  public maximumTimeBetweenRaycasts = 1000; // For mouse wheel event
+  public mouseDistanceThresholdBetweenRaycasts = 5; // For mouse wheel event
```

Comment:
I don't think these should be exposed. But we can keep them while you experiment